src/main/java/com/ericsson/ei/frontend/EIRequestsController.java

```diff
@@ -48,8 +52,8 @@
 public class EIRequestsController {
 
     private static final Logger LOG = LoggerFactory.getLogger(EIRequestsController.class);
-
-    private CloseableHttpClient client = HttpClientBuilder.create().build();
+    private static final String X_PATH_NAME = "x-auth-token";
```

Comment:
Couldn't this simply be called X_AUTH or X_AUTH_TOKEN?

Reply:
Is "x-auth-token" HTTP Standard?
I think also that we should have "auth" in the varaible name.

Reply:
No, it's an unregistered header and is missing here: https://www.iana.org/assignments/message-headers/message-headers.xhtml.
It's used in spring for session authentication and every other app can have their own definition of its use.

Reply:
Fixed.